scripts/lambda-build/create-package.sh

```diff
@@ -2,6 +2,21 @@
 
 echo "Executing create_package.sh..."
 
+# Check Python version matches runtime
+python_minor_version="$(python3 --version)"
+python_version="${python_minor_version%.*}"
+prefix="Python "
+local_version="${python_version#$prefix}"
+runtime_prefix="python"
+lambda_version="${runtime#$runtime_prefix}"
+
+if [ "$lambda_version" != "$local_version" ]; then
+  echo "Error: local Python version does not match Lambda Python runtime"
+  echo "Local Python version: $local_version"
+  echo "Lambda Python version: $lambda_version"
+  exit 1
+fi
```

Comment:
Regarding DRY principal, Is there any way to encapsulate this if block in a function (or similar) so both scripts can share the same logic? Otherwise if you need to update one block, you'll have to update the other.

Reply:
good idea, this should be possible